VmAgent.Core/Interfaces/ProcessRunner.cs

```diff
@@ -79,6 +82,27 @@ public override Task CollectLogs(string processId, string logsFolder, ISessionHo
             return Task.CompletedTask;
         }
 
+        public override Task CreateStartGameExceptionLogs(string logsFolder, string exceptionMessage)
+        {
+            try
+            {
+                _logger.LogVerbose("Collecting logs for failed start game process.");
+                string destinationFileName = Path.Combine(logsFolder, ConsoleLogCaptureFileName);
+
+                using (StreamWriter sw = File.AppendText(destinationFileName))
+                {
+                    sw.WriteLine(DateTime.Now.ToString("HH:mm:ss D"));
```

Comment:
I see the D as a character in the log file as below -
```
20:27:14 0:D
System.ComponentModel.Win32Exception (2): The system cannot find the file specified.
   at System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
```
I think we can use (DateTime.Now.ToString("yyyy-MM-dd'T'HH:mm:ss.fffK") to match the date format in docker. 

Here is the example of logs from docker when it throws exception:
```
{"log":"Running Startup Script\n","stream":"stdout","time":"2022-06-15T21:37:43.0271122Z"}
{"log":"Certificates located at: C:\\GameCertificates\n","stream":"stdout","time":"2022-06-15T21:37:43.0481836Z"}
{"log":"'C:\\Assets\\wrappser.exe' is not recognized as an internal or external command,\r\n","stream":"stderr","time":"2022-06-15T21:37:43.25104Z"}
{"log":"operable program or batch file.\r\n","stream":"stderr","time":"2022-06-15T21:37:43.25104Z"}

```